src/utils/components/HorizontalNavbar/HorizontalNavbar.tsx

```diff
@@ -15,15 +15,17 @@ import { NavPageKubevirt, trimLastHistoryPath } from './utils/utils';
 import './horizontal-nav-bar.scss';
 
 type HorizontalNavbarProps = {
-  expandedSpecLoading?: boolean;
+  error?: any;
```

Comment:
Is there any way to narrow this type?

Reply:
no. The k8sWatch hook use any for the error type so nothing to do about it.